### ISSUE TITLE
Hidden Text Button

### ISSUE THREAD

**Issue by srph** [dev-onenetworkecommerce/aia#52]
Like [PrimerCSS's](http://primercss.io/buttons/#hidden-text-button)

**Comment by JefMari** [comment]
rather than button, should this be ok if it'll be done like this [Primer css Truncation](http://primercss.io/utilities/#truncation) ?

**Comment by srph** [comment]
We already have that utility, AFAICR.

We need this because it indicates "click-me-to-read-more".

**Comment by JefMari** [comment]
ohh i see

**Comment by srph** [comment]
Here's the link to [Text Ellipsis](https://rawgit.com/dev-onenetworkecommerce/aia/master/docs/index.html#/utilities?_k=h6ac7b).

**Comment by JefMari** [comment]
ahh ok.
i [Primer css Truncate](http://primercss.io/utilities/#truncation) it has a variant of `expandable`, which makes the shortened text expand when hover. will we need that or not?

**Comment by srph** [comment]
Good catch. But I think that should be a new feature (separate issue). Can you make one? haha

**Comment by JefMari** [comment]
Yeah sure :100:
